### PR TITLE
Create Account and Vendor data containers

### PR DESCRIPTION
**Pull request by cxfloeder**
Create Account and Vendor classes that contain all the designated fields for a Vendor and Account.

### CODE REVIEW

File: src/java/data/Vendor.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.data;
+
+import java.util.*;
+
+/**
+ * A class representing a billing vendor object.
```
Comment (jakecarlson1) [inline src/java/data/Vendor.java:19]: Nit: this javadoc can be on a single line.

File: src/java/data/Account.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.data;
+
+/**
+ * A class representing a billing account object.
```
Comment (jakecarlson1) [inline src/java/data/Account.java:17]: Nit: this javadoc can be on a single line

**Comment by cxfloeder** [comment]
Changed the file structure a bit, so it hopefully won't ruin anybody else's code. Passed the initial unit tests. Let me know if I need to add more for right now.

File: src/java/data/Account.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        this.entity = entity;
+        this.currency = currency;
+        this.direction = direction;
+        this.legacyAccountID = legacyAccountID;
+        this.nextGenAccountID = nextGenAccountID;
+        this.matchingMode = matchingMode;
+        this.agregationMode = aggregationMode;
+    }
+
+    // Create getter and setter methods for each field.
```
Comment (ianjjohnson) [inline src/java/data/Account.java:41]: maybe this was meant as a TODO? probably just remove it.

File: src/java/data/Vendor.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    public Vendor(String vendorID, String legacyVendorID, int nextGenVendorID) {
+        this.vendorID = vendorID;
+        this.legacyVendorID = legacyVendorID;
+        this.nextGenVendorID = nextGenVendorID;
+        accountList = new ArrayList<Account>();
+    }
+
+    public void addAccount(Account account) {
```
Comment (ianjjohnson) [inline src/java/data/Vendor.java:36]: nit: maybe addAccount and getAccounts should be next to eachother?

File: src/java/data/Vendor.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    public void setLegacyVendorID(String legacyVendorID) {
+        this.legacyVendorID = legacyVendorID;
+    }
+
+    public int getNextGenVendorID() {
+        return nextGenVendorID;
+    }
+ 
+    public void setNextGenVendorID(int nextGenVendorID) {
+        this. nextGenVendorID = nextGenVendorID;
```
Comment (ianjjohnson) [inline src/java/data/Vendor.java:62]: nit: remove extra space after the period

File: src/main/java/data/Account.java [hunk 6]
```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Google LLC
```
Comment (ianjjohnson) [inline src/main/java/data/Account.java:1]: Why is this exact same Account class in the code twice?
Only one copy of it should exist

Comment (ianjjohnson) [reply src/main/java/data/Account.java:1]: Looks like this is actually the case for all of the code -- it's in here twice?

Comment (cxfloeder) [reply src/main/java/data/Account.java:1]: Something must have happened when I was changing the file structure that duplicated the files. I will fix that up soon.

**Pull request merged**